lib/kms_monitor/cloudwatch.rb

```diff
@@ -105,7 +105,7 @@ def update_db_record(dbrecord, kmsevent)
       end
 
       table_name = ENV.fetch('DDB_TABLE')
-      ttl = Time.now.utc + ENV.fetch('RETENTION_DAYS')
+      ttl = Time.now.utc + ENV.fetch('RETENTION_DAYS').to_i
```

Comment:
Prefer `Integer(foo)` over `foo.to_i`. The former raises `ArgumentError` if given garbage input, whereas `String#to_i` will swallow errors and return `0`.

Reply:
Done.